### ISSUE TITLE
Version info

### ISSUE THREAD

**Issue by lkatalin** [keylime/rust-keylime#313]
Do we need to implement this also? https://github.com/keylime/keylime/pull/860

**Comment by THS-on** [comment]
Yes, because it makes the life for the tenant easier to determine what version it should use for API interactions.

**Comment by lkatalin** [comment]
Thank you @THS-on !
